태영/G3_1865_웜홀.py

```diff
@@ -0,0 +1,37 @@
+import sys
+input = sys.stdin.readline
+
+tc = int(input())
+
+
+def bf():
+    for i in range(n):
+        for j in range(len(edges)):
+            cur = edges[j][0]
+            next_node = edges[j][1]
+            cost = edges[j][2]
```

Comment:
```suggestion
            cur, next_node, cost = edges[j]
```

요건 어때요??

(그냥 진짜 조금이라도 줄여 보자 하는 취향 차이,,,)